bodhi/tests/server/functional/test_generic.py

```diff
@@ -111,18 +111,49 @@ def test_markdown(self):
         res = self.app.get('/markdown', {'text': 'wat'}, status=200)
         self.assertEquals(
             res.json_body['html'],
-            "<div class='markdown'>"
-            "<p>wat</p>"
-            "</div>"
+            '<div class="markdown">'
+            '<p>wat</p>'
+            '</div>'
```

Comment:
Several changes in this pull request swap quote styles like this. The change is fine, but it'd be best if it was done in an atomic commit that was separate of the intended fix.